packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -15,7 +15,7 @@ import { useQueryString } from './hooks/useQueryString';
 import { useFloatingSuggestionItems } from './hooks/useFloatingSuggestionItems';
 import { useSelectedItems } from './hooks/useSelectedItems';
 import { IFloatingSuggestionItemProps } from '../../FloatingSuggestionsComposite';
-import { copyToClipboard } from '@uifabric/experiments/src/components/SelectedItemsList/utils/copyToClipboard';
+import { copyToClipboard } from '../SelectedItemsList';
```

Comment:
Due to the setup of some consumers of this repo you have to explicitly import from the index file:
```suggestion
import { copyToClipboard } from '../SelectedItemsList/index';
```